devel/prest/Portfile

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/prest/prest 1.0.8 v
+categories          devel databases
+license             MIT
+maintainers         {outlook.de:judaew @judaew} openmaintainer
+
+description         pREST (PostgreSQL REST) is REST API for any Postgres database.
+long_description    \
+    pREST (PostgreSQL REST), simplify and accelerate development, instant, \
+    realtime, high-performance on any Postgres application, existing or new
+homepage            https://prestd.com
+
+checksums           ${distname}${extract.suffix} \
+                        rmd160  448a195892366ae2fd5a07888a9c3bb285ffee4f \
+                        sha256  0751ceb27bf9e32cbc67a6e01f657b9532b9eac6aa6198f1929da2c56a3316c1 \
+                        size    97783
+
+build.args          "-o prestd cmd/prestd/main.go"
```

Comment:
You can drop the double quotes, and you don't have to specify the specific Go file as the build target.

The following should work:
```
build.args            -o prestd ./cmd/prestd
```

Reply:
Thanks 🌟